src/components/ui/image.tsx

```diff
@@ -22,6 +22,7 @@ const Component = (
                 loaded ? 'opacity-100' : 'opacity-0',
                 !transitionDisabled && '!transition',
                 className,
+                'sharpness-fix',
```

Comment:
Just use tailwindcss, no need to add it to the global CSS file. And move it to top of all classnames
```suggestion
		'[overflow-clip-margin:unset]',
                loaded ? 'opacity-100' : 'opacity-0',
                !transitionDisabled && '!transition',
                className,
```

Reply:
Use `content-fit: fill` css property

Reply:
Why? It will just stretch the image

Reply:
I don't know why, but it changes behavior of image resizing algorithm and this actually solves the problem. Also, this is much cleaner solution than using unrelated css property